src/test/java/gov/healthit/chpl/aqa/asserts/ViewVersionPageAsserts.java

```diff
@@ -59,6 +61,7 @@ public void iSeeMessageInVersionsToMergeSection(final String expectedText) {
     @Then("^I see error message \"(.*)\" for version \"(.*)\"$")
     public void iSeeFieldIsRequiredError(final String expectedErrorMessage, final String versionId) {
         String actualErrorMessage = ViewVersionPage.versionMissingErrorMessage(getDriver(), versionId).getText();
+        ((JavascriptExecutor) getDriver()).executeScript("arguments[0];", actualErrorMessage);
```

Comment:
What does this do? If it's trying to execute a script, what script? This doesn't make any sense to me